src/modules/attributes.ts

```diff
@@ -26,28 +26,31 @@ function updateAttrs(oldVnode: VNode, vnode: VNode): void {
   if (oldAttrs === attrs) return;
   oldAttrs = oldAttrs || {};
   attrs = attrs || {};
+  const lowAttrs: any = {};
 
   // update modified attributes, add new attributes
   for (key in attrs) {
     cur = attrs[key];
-    old = oldAttrs[key];
+    const lowkey = key.toLowerCase();
+    lowAttrs[lowkey] = cur;
+    old = oldAttrs[lowkey];
     if (old !== cur) {
```

Comment:
For subsequent patchings (a VNode against a VNode), won't that compare the lowerCased current attribute with the non lowerCased previous attribute and thus do a wasteful setAttribute() ?